frontend/validators.js

```diff
@@ -44,7 +46,10 @@ const feeValidator = (sendAmount, transactionFee, balance) => {
 const mnemonicValidator = (mnemonic) => {
   let validationError = null
 
-  if (!validateMnemonic(mnemonic)) {
+  if (
+    !validateMnemonic(mnemonic) ||
+    (isPaperWalletMnemonic(mnemonic) && !decodePaperWalletMnemonic(mnemonic))
```

Comment:
maybe would be cleaner to rename `isPaperWalletMnemonic` to something like `isMnemonicInPaperWalletFormat` and create function `validatePaperWalletMnemonic` which would check `isMnemonicInPaperWalletFormat` and would try to `decodePaperWalletMnemonic` so we end up with `!validateMnemonic(mnemonic) && !validatePaperWalletMnemonic(mnemonic)`